src/application-error-boundary/ApplicationErrorBoundary.jsx

```diff
@@ -92,10 +95,12 @@ class ApplicationErrorBoundary extends React.Component {
       const errorDetails = activeError.message.toString();
       const errorText = intl.formatMessage({ id: 'terraApplication.errorBoundary.defaultErrorMessage' }, { errorDetails });
       return (
-        <StatusView
-          variant="error"
-          message={errorText}
-        />
+        <div role="alert" id="applicationErrorBoundary" tabIndex="-1">
```

Comment:
An `id` is probably safe, but using a `ref` would be more safe.

Reply:
Switched to `ref`.